tests/integration/raw.rs

```diff
@@ -0,0 +1,82 @@
+// Copyright 2019 TiKV Project Authors. Licensed under Apache-2.0.
+
+use crate::{arb_batch, integration::pd_addr};
+use futures::executor::block_on;
+use proptest::{arbitrary::any, proptest};
+use tikv_client::{raw::Client, Config, KvPair, Value};
+
+proptest! {
+    #[test]
+    fn point(
+        pair in any::<KvPair>(),
+    ) {
+        let client = block_on(Client::connect(Config::new(pd_addr()))).unwrap();
+
+        block_on(
+            client.put(pair.key().clone(), pair.value().clone())
+        ).unwrap();
```

Comment:
I would use `await` rather than `block_on` since the latter does not permit concurrency

Reply:
I would too!

But, the `proptest` macro isn't compatible with async/await, I tried using a bare approach with `async` test functions but since the `TestRunner` operates in a closure you can't use `.await` or `?` anyways. 

I'm considering trying to patch this upstream, but in the meantime, this seems easiest as it avoids a fair bit of testrunner/handling boilerplate.

Reply:
I was thinking you would just use another function and call that from the test function, but if you think that is not worth the boilerplate, then we can keep things like this